setup.py

```diff
@@ -55,6 +55,7 @@
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
is there a reason not to go ahead and add 3.9 as well? (also to tests)

Reply:
Good call: https://github.com/googleapis/python-spanner-django/pull/568.